src/platform/windows/events_loop.rs

```diff
@@ -295,8 +296,13 @@ lazy_static! {
 thread_local!(static CONTEXT_STASH: RefCell<Option<ThreadLocalData>> = RefCell::new(None));
 struct ThreadLocalData {
     sender: mpsc::Sender<Event>,
+// <<<<<<< HEAD
```

Comment:
Please remove that!

Reply:
Whoops, accidentally left those in. Those should be gone now.